cfme/common/vm.py

```diff
@@ -316,8 +316,7 @@ def find_quadicon(
             else:
                 raise TemplateNotFound("Template '{}' not found in UI!".format(self.name))
 
-        # this is causing some issues in 5.5.0.9, commenting out for a bit
-        # paginator.results_per_page(1000)
+        paginator.results_per_page(1000)
```

Comment:
As discussed on IRC - I believe this is acceptable, even preferred actually, outside of tests. We don't target pagination testing here, we just want to get our quadicon, so I see no issue with this. However, we should have pagination tests for all the pages with paginators on them where we test going forward, backwards + different amount per page. Not here though.